common-lib/src/cmr/common/cache/in_memory_cache.clj

```diff
@@ -6,12 +6,39 @@
   (:require
    [clojure.core.cache :as cc :refer [defcache]]
    [cmr.common.cache :as c]
-   [cmr.common.log :as log :refer (debug info warn error)]
+   [cmr.common.log :refer [warn]]
    [cmr.common.time-keeper :as time-keeper]
    [cmr.common.dev.record-pretty-printer :as record-pretty-printer])
   (:import
    (clojure.core.cache CacheProtocol)))
 
+(defmulti size-in-bytes class)
+
+(defmethod size-in-bytes :default
```

Comment:
I am afraid this default implementation would be off by too much for UMM JSON or JSON data.

Reply:
Added tests for various JSON representations as string and in EDN structures.